omnibus/package-scripts/chef-workstation/postinst

```diff
@@ -25,7 +43,39 @@ else
     PREFIX="/usr"
 fi
 
-binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client"
+binaries="chef-run berks chef chef-cli chef-analyze chef-apply chef-shell chef-solo chef-vault cookstyle delivery foodcritic inspec kitchen knife ohai push-apply pushy-client pushy-service-manager chef-client hab"
+
+hab_target_path="$PREFIX/bin/hab"
+if [ -L "$hab_target_path" ]; then
+  loc=$(readlink "$hab_target_path")
+  case $loc in
+    *chef-workstation*)
+      # We own this path, so we're not taking it over from another install.
+      ;;
+    *)
+      takeover="1"
+  esac
+elif [ -f "$hab_target_path" ]; then
+  takeover="1"
+  rename="1"
+fi
+
+if [ "$takeover" = "1" ]; then
+  rm -rf "$hab_target_path"
```

Comment:
```suggestion
```

What do you think about, in the symlink case, not doing anything because it will be overwritten by the `ln -sf` below?